### DISCUSSION TITLE
Is it possible to customize LED on Tasmota's S26

### DISCUSSION THREAD

**Discussion by guydvir2** [arendst/Tasmota#11340] [Q&A] [answered]
Hi,

I wish to use this plug, with multi-press functionality ( different timeouts ).
Is there a way to define that is I press the button for 3 time, LED will blink in a certain manner ? or perhaps if it is in some timeout mode, LED will flash, while when it is constantly ON LED will be ON ( no flashing ) ?

Guy

**Comment by sfromis** [comment, marked as answer]
As the template for [Sonoff S26](https://templates.blakadder.com/sonoff_S26.html) includes a Led1i pin, you can do a lot of configuration. Tasmota has some features for customizing [status leds](https://tasmota.github.io/docs/Lights/#status-leds), but you can also configure it as a relay for PWM output, giving you full control over when what happens. May take some learning curve to get right. As Tasmota has features for things like blinking, you many not need timers to control all details.

**Reply by guydvir2** [reply]
@sfromis Can you be more specific, some example or any reference for such cases ?

**Reply by sfromis** [reply]
If you config it as a Relay, you can use commands like `Power2 on` to control it. If you config it as a PWM, you can additionally use the Dimmer command. Read about rules (be prepared for a bit of learning curve) to automate the actions.
